repository/src/main/java/org/alfresco/repo/content/metadata/AbstractMappingMetadataExtracter.java

```diff
@@ -1462,7 +1463,7 @@ public void embed(
      * @param systemMetadata   Metadata keyed by system properties
      * @return              Returns the metadata keyed by the content file metadata properties
      */
-    private Map<String, Serializable> mapSystemToRaw(Map<QName, Serializable> systemMetadata)

```

Comment:
Made protected so it could be overridden in AsynchronousExtractor, where we wish to map to fully qualified QNames name rather than the raw name.